src/Orchard.Web/Modules/Orchard.Layouts/Controllers/LayoutController.cs

```diff
@@ -15,15 +16,29 @@ public class LayoutController : Controller {
         private readonly ILayoutManager _layoutManager;
         private readonly ILayoutModelMapper _mapper;
 
-        public LayoutController(IContentManager contentManager, ILayoutManager layoutManager, ILayoutModelMapper mapper) {
+        public LayoutController(
+            IContentManager contentManager, 
+            ILayoutManager layoutManager, 
+            ILayoutModelMapper mapper,
+            IOrchardServices orchardServices) {
 
             _contentManager = contentManager;
             _layoutManager = layoutManager;
             _mapper = mapper;
+            Services = orchardServices;
+
+            T = NullLocalizer.Instance;
         }
 
+        public IOrchardServices Services { get; set; }
+        public Localizer T { get; set; }
+
         [HttpPost, ValidateInput(enableValidation: false)]
-        public ContentResult ApplyTemplate(int? templateId = null, string layoutData = null, int? contentId = null, string contentType = null) {
+        public ActionResult ApplyTemplate(int? templateId = null, string layoutData = null, int? contentId = null, string contentType = null) {
+            if(!Services.Authorizer.Authorize(Permissions.ManageLayouts, T("Not authorized to manage layouts."))) {
```

Comment:
Managing template layouts or applying them are two different things. Perhaps as a content editor I should be able to use existing templates, but not create or edit them.